Controller/Adminhtml/CompanyCodes/Get.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2018 Classy Llama Studios, LLC
+ * @author      sean.templeton
+ */
+
+namespace ClassyLlama\AvaTax\Controller\Adminhtml\CompanyCodes;
+
+use ClassyLlama\AvaTax\Exception\AvataxConnectionException;
+use Magento\Framework\DataObject;
+
+class Get extends \Magento\Backend\App\Action
+{
+    protected $resultPageFactory;
+
+    /**
+     * @var \ClassyLlama\AvaTax\Framework\Interaction\Rest\Company
+     */
+    protected $company;
+
+    /**
+     * @var \ClassyLlama\AvaTax\Helper\Config
+     */
+    private $config;
+
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Framework\Controller\Result\JsonFactory $resultPageFactory,
+        \ClassyLlama\AvaTax\Framework\Interaction\Rest\Company $company,
+        \ClassyLlama\AvaTax\Helper\Config $config
+    )
+    {
+        parent::__construct( $context );
+        $this->resultPageFactory = $resultPageFactory;
+        $this->company = $company;
+        $this->config = $config;
+    }
+
+    public function execute()
+    {
+        $companies = [];
+        $postValue = $this->getRequest()->getPostValue();
+        $isProduction = (bool)$this->getRequest()->getParam( 'mode' );
+        $resultJson = $this->resultPageFactory->create();
+        $scope = isset( $postValue['scope'] ) ? $postValue['scope'] : null;
+        $scopeType = $postValue['scope_type'] === 'global' ? \Magento\Store\Model\ScopeInterface::SCOPE_STORE : $postValue['scope_type'];
+        $currentCompanyId = $this->config->getCompanyId( $scope, $scopeType, $isProduction );
+
+        try
```

Comment:
@LordZardeck - For try/catch blocks, IF blocks, and loops, our coding standard is for opening braces to be on the same line as the opening construct.  ("catch" should also be on the same line as the closing brace of the "try".) . Again, this is for future reference only, but this one is a little more important than the issue with spaces inside parens and should definitely be followed.